frontend/src/pages/PassingTest/QuestionItem/types.ts

```diff
@@ -0,0 +1,11 @@
+import { IQuestionObject } from 'types/test';
+
+export interface IQuestionProps {
+  handleChange: (qN: number) => (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: {
+    [key: number]: number;

```

Comment:
is it write? always in objects the key it is string
```suggestion
    [key: string]: number;

```